posthog/settings.py

```diff
@@ -167,6 +167,14 @@ def print_warning(warning_lines: Sequence[str]):
 KAFKA_HOSTS = ",".join(KAFKA_HOSTS_LIST)
 KAFKA_BASE64_KEYS = get_from_env("KAFKA_BASE64_KEYS", False, type_cast=strtobool)
 
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
```

Comment:
printer go brrrrrrrrr

Reply:
This will be fixed before it goes out I promise. PR wasn't meant to be merged.